test/test.js

```diff
@@ -25,12 +25,13 @@ describe('ipfs node api', function () {
     ipfsd.disposable(function (err, node) {
       if (err) throw err
       log('ipfs init done')
+      ipfsNode = node
 
-      node.startDaemon(function (err, ignore) {
+      ipfsNode.startDaemon(function (err, ignore) {
         if (err) throw err
         log('ipfs daemon running')
 
-        ipfs = ipfsApi(node.apiAddr)
+        ipfs = ipfsApi(ipfsNode.apiAddr)
```

Comment:
Wanted a reference to the node so I could stop the daemon and check for the error in the last test. I can revert back to using `node.*` instead of `ipfsNode.*` for this before() block.. I thought it was more readable to stick with one variable name